CHANGELOG.md

```diff
@@ -2,10 +2,12 @@
 
 ## Unreleased
 
+- added: Support for splitting LTC to BTC wallets for recovering BTC erroneously sent to an LTC address
 - added: "Force Light Account Creation" developer mode setting
 - changed: Light account backup notification card now persists no matter what while logged in to a light account
 - changed: Replaced 'react-native-camera' with 'react-native-vision-camera'
 - fixed: Stabilize the account sync bar progress motion.
+- fixed: Possible to do invalid split operations from a BTC wallet
```

Comment:
Since the regression is in develop/staging and not production, we don't need a changelog entry for it